packages/api/src/schema/api-schema.yaml

```diff
@@ -228,32 +228,46 @@ components:
         webhookId:
           readOnly: true
           type: string
-        eventId:
+          description: ID of the webhook this request was made for
+        event:
           readOnly: true
           type: string
+          description: The event type that triggered the webhook request
         createdAt:
           readOnly: true
           type: number
           description: |
-            Timestamp (in milliseconds) at which webhook response object was
+            Timestamp (in milliseconds) at which webhook request object was
             created
           example: 1587667174725
         duration:
           type: number
+          description: The time taken to make the webhook request
         request:
           type: object
+          properties:
```

Comment:
Should we include the URL here too, in case the webhook config changes (I think we talked about this on Discord, don't remember the decision)

Reply:
Oh yep good point, also the secret?

Reply:
ah yep forgot i already looked at that, we have the sig in the headers